check.py

```diff
@@ -107,6 +107,10 @@
             cmd += ['-O0'] # test that -O0 does nothing
         else:
           cmd += ['-O']
+        if 'PRINT_FULL' in asm:
```

Comment:
Fair point, I wanted to keep my options open ;) In case we do want multiple files, this would mean we don't need to change anything.

Anyhow, I don't feel strongly about any of it. If you do, let me know what you'd prefer.